client/asset/btc/btc.go

```diff
@@ -312,6 +312,9 @@ type BTCCloneCFG struct {
 	// ZECStyleBalance is for clones that don't support getbalances or
 	// walletinfo, and don't take an account name argument.
 	ZECStyleBalance bool
+	// ZECUnifiedAddrs is for ZCash, which doesn't support getnewaddress, and
+	// has to extract the transparent address from a unified address.
+	ZECUnifiedAddrs bool
```

Comment:
Looks to resolve https://github.com/decred/dcrdex/issues/2134

Reply:
I edited this into the PR description, but you must have already been on the page.